ios/Classes/BetterPlayerPlugin.m

```diff
@@ -193,18 +193,40 @@ - (void) setupRemoteCommands:(BetterPlayer*)player isExtraVideo:(BOOL)isExtraVid
 }
 
 - (void) setupRemoteCommandNotification:(BetterPlayer*)player, NSString* title, NSString* author , NSString* imageUrl, BOOL isLiveStream {
-    float positionInSeconds = player.position /1000;
-    float durationInSeconds = player.duration/ 1000;
-
+    // This function is always called double times at the end of video due to the default behavior of AVPlayer
+    // This check is used to prevent the latest call.
+    if (player.position >= player.duration - 500) {
+        return;
+    }
+    float positionInSeconds = player.position / 1000;
```

Comment:
Sorry, please re-check.

I think, the result of this calculation round down to int. ( `int64_t` / `int` )
We may need `F` like `1000F`.

(I worry about it is the reason of causing "-0:01".)

Reply:
@noboru-i 
I just test your case,

> I think, the result of this calculation round down to int. ( int64_t / int )

Yes, the result got rounded to `int`

> We may need F like 1000F.

i got the `float` value by using `(float)1000`

> (I worry about it is the reason of causing "-0:01".)
eg: Duration: `32.000000` (the duration always got round in all videos)

- If the last position = `31.768002` (the Control center will round it to `32.000000`) ==> result is `-0:00`

- If the last position = `31.235000` (the Control center will round it to `31.000000`) ==> result is `-0:01` ==> NG

Reply:
We can't round up all the `position` because the `position` which got called every second will also be rounded up (those `position` are not `integer`)
![image](https://github.com/dwango-nfc/betterplayer/assets/100773699/d2e2f3d7-ca1d-400b-bc6c-fcbcb69c0e50)